app/models/project.rb

```diff
@@ -91,6 +91,17 @@ class Project < ApplicationRecord
 
   scope :by_language, lambda { |languages|
     joins(:language).where(languages: { name: languages })
+  scope :with_no_cc, lambda {
+    left_joins(:code_climate_project_metric).where(code_climate_project_metrics: { id: nil })
+  }
+
+  scope :with_no_cc_rate, lambda {
+    left_joins(:code_climate_project_metric)
+      .where(code_climate_project_metrics: { code_climate_rate: nil })
+  }
+
+  scope :with_no_cc_nor_cc_rate, lambda {
```

Comment:
I might be wrong but I interpret `nor` in `with_no_cc_nor_cc_rate` as "projects without cc AND without cc rate", and `with_no_cc_or_cc_rate` as "projects without cc OR without cc rate",

Reply:
This was the only way I could make the OR work for scopes. I can change it to `without_cc_or_cc_rate`